### PR TITLE
Bugfix/species filter is getting ignored in the metadata search

### PR DESCRIPTION
**Pull request by upendrakumbham**
This is the backend PR for the integration of front-end PR - [front-end](https://github.com/ebi-gene-expression-group/atlas-components/pull/166)

This PR integrates recently published components to get changes to modified components into SCXA.

### CODE REVIEW

**Review by ke4 — DISMISSED** [review]
LGTM!

**Comment by lingyun1010** [comment]
I am not in the review list, so I will just leave my commets here.
You deleted some package-log.json files again, and your commits are unverified, and the jenkins are not passed. Please fix these. Thanks.

File: app/src/main/javascript/bundles/cell-type-wheel-heatmap/index.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ import CellTypeWheelMultiExperimentHeatmap from '@ebi-gene-expression-group/scxa
 
 const render = (options, target) => {
   ReactDOM.render(
-    <CellTypeWheelMultiExperimentHeatmap {...options} />,
+    <CellTypeWheelMultiExperimentHeatmap {...options}
+    />,
```
Comment (lingyun1010) [inline app/src/main/javascript/bundles/cell-type-wheel-heatmap/index.js]: We don't need a separate line here I think. Just format issue though.

**Review by lingyun1010 — CHANGES_REQUESTED** [review]
There are still some `package-lock.json` been deleted. Could you pleaes fix that before merging? Thanks.

File: app/src/main/javascript/bundles/cell-type-wheel-heatmap/index.js (hunk 1 above)
Comment (upendrakumbham) [reply app/src/main/javascript/bundles/cell-type-wheel-heatmap/index.js]: Fixed it

**Review by lingyun1010 — DISMISSED** [review]
Normally, we have a space between `}` and `/>` in jsp, I was trying to make a commit suggestion but just committed straight away. LGTM! 
Besides, some of your commits are not `verified` please figure it out too.

**Pull request merged**